scripts/adapter/ua.js

```diff
@@ -44,12 +44,14 @@ var uaTrackingJS = function(trackingJSOptions, trackingJSHelper) {
      * includes the ua analytics js
      */
     this.appendAnalyticsJs = function () {
-        /* jshint ignore:start */
-        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-            (i[r].q=i[r].q||[]).push(arguments);},i[r].l=1*new Date();a=s.createElement(o),
-            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m);
-        })(window,document,'script','//www.google-analytics.com/analytics.js','ga'); 
-        /* jshint ignore:end */
+        if (typeof(ga) === 'undefined') {
```

Comment:
in this case i also receive this error:
`Uncaught ReferenceError: ga is not defined`
so i think that my way is the correct one ;)